.github/workflows/continuous-integration.yml

```diff
@@ -374,3 +374,43 @@ jobs:
 
       - name: "BC Checks"
         run: tools/vendor/bin/roave-backward-compatibility-check --from=origin/${{ github.base_ref }}
+
+  composer-depedency-analyser:
```

Comment:
Since the dependency set is tiny, let's please drop this as well, as part of the change:

https://github.com/Roave/BetterReflection/blob/ee1ce025762a8784c89e0cb4ef59e85fecbd94ac/composer.json#L5-L11

Practically, just `composer.json` and `composer.lock` should change in this PR

Reply:
Sure, do you want me to move those changes to separate MR?

Reply:
I'd just push to the same PR - I don't really want to add additional tooling for this edge case though, since dependency removal is not really such a frequent detail (happened in the last major version only)

Reply:
The tool actually detects much more, just this repo does not really suffer with other issues :) Ok then.

Reply:
Yeah, but I do kinda know this stuff, as I help maintaining https://github.com/maglnet/ComposerRequireChecker ;-)

Reply:
Yeah I know, those tools compete now :) 

We built this one just because ComposerRequireChecker was taking too much time in our CI. But we were using it successfully for a long time, thanks for it!